subprojects/docs/src/docs/release/notes.md

```diff
@@ -399,6 +403,11 @@ A new behavior that made its evaluation eager (like for any other block) was int
 Now, the old behavior has been removed and switching on the new one is no longer necessary. 
 If you need to defer evaluation, please use `afterEvaluate {}`.
 
+### Annotation processors on the compile classpath are now ignored
+
+Annotation processors on the compile classpath are no longer detected and used when compiling Java projects.
+Please add them to the [annotation processor path](userguide/java_plugin.html#example_declaring_annotation_processors) instead.
+
```

Comment:
I think it would be helpful to include some words here about what kind of impact this may have.  Someone upgrading from 4.x to 5.0 might not look at this when they're having compile failures.